consumer-rest-core/src/main/scala/za/co/absa/spline/consumer/rest/controller/DataSourcesController.scala

```diff
@@ -68,8 +68,8 @@ class DataSourcesController @Autowired()(
     @ApiParam(value = "Text to filter the results")
     @RequestParam(value = "searchTerm", required = false) searchTerm: String,
 
-    @ApiParam(value = "Write mode (true - append, false - overwrite")
-    @RequestParam(value = "append", required = false) append: java.lang.Boolean,
+    @ApiParam(value = "Write mode. Comma separated values of: `true` - append, `false` - overwrite, empty string - read only")
+    @RequestParam(value = "append", required = false) append: String,
```

Comment:
It seems weird that `append = ""` means that it's read. Wouldn't it be better to have three booleans for each or something like enum `APPEND`, `OVERWRITE` and `READ`?

Reply:
This API is a subject for a redesign, I don't want to spend much time on it.

Reply:
ok